### ISSUE TITLE
JAWS is not Reading Table Rows as clickable properties.

### ISSUE THREAD

**Issue by bserrano1** [GSA/sam-design-system#1155]
**Components Involved**
Please list the common component(s) involved that you're experiencing an issue with
Table Component -Highlighting and click event
https://cg-fa19003e-5296-4960-ac2e-caccfeb620ac.sites.pages.cloud.gov/site/gsa/sam-design-system/documentation/components/table/examples 

**Expected Behavior**
How you expect the listed component(s) to function
JAWS should be reading each row as a button

**Actual Behavior**
How the listed component(s) actually function
JAWS skips over the table entirely

**Your Environment**
Angular Version: X.Y.Z
@gsa-sam packages version: X.Y.Z (Please list individually if all packages are not same version)
Angular Version: "^13.3.11"
GSA Packages:
@gsa-sam/components@13.0.5
@gsa-sam/sam-material-extensions@13.0.5
@gsa-sam/sam-formly@13.0.5
@gsa-sam/layout@13.0.3
@gsa-sam/layouts@13.0.0-beta-0
@gsa-sam/ngx-uswds@13.0.0-beta.1
@gsa-sam/ngx-uswds-icons@13.0.0-beta.2
@gsa-sam/sam-styles@0.0.150
@gsa-sam/shared@13.1.10

**Ticket Number**
Please provide your JIRA ticket number that is dependent on this fix. If there is none, please create
one to validate the issue once the fix has been released. DO NOT provide the url, please ONLY
provide the ticket number.
IAE-XYZ
https://cm-jira.usa.gov/browse/IAEMOD-12366

**Additional context**
Screenshots / recordings, minimal reproduction on stackblitz, steps to reproduce, code snippets, etc.
![image](https://user-images.githubusercontent.com/26351294/230650339-df97bdf7-d52c-402e-81a2-9c4efbc34b6d.png)

**Comment by cwolf10** [comment]
Hey @bserrano1 ,

I've gotten a chance to look into this issue further. The SDS table currently aligns with the Material table for this exact scenario. https://material.angular.io/components/table/examples#table-row-binding.

Further, any proposed changes would need to be done on the MFE side because the result of clicking a row, and therefore the appropriate semantic and text labeling, is specific to the implementing MFE.

If there is a specific component change that can be made to support a solution for your particular use-case, please open a feature request and we will see what we can do.
